src/main/java/seedu/address/model/widget/WidgetObject.java

```diff
@@ -112,31 +119,22 @@ public String textTwo() {
         return divs[4] == null ? "" : divs[4];
     }
 
-    /**
-     * Getter for 'attributes'.
-     *
-     * @return Div3.
-     */
-    public String divThree() {
-        return divs[5] == null ? "" : divs[5];
-    }
-
     /**
      * Getter for 'attributes'.
      *
      * @return Text3.
      */
     public String textThree() {
-        return divs[6] == null ? "" : divs[6];
+        return divs[5] == null ? "" : divs[5];
```

Comment:
Might want to extract out magic numbers as constants so smth like `TEXT_THREE_POS = 5`, `DIV_THREE_POS = 6`